PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ConnectionStringCheck.java

```diff
@@ -0,0 +1,109 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.LocalInspectionTool;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.psi.JavaElementVisitor;
+import com.intellij.psi.PsiClass;
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiElementVisitor;
+import com.intellij.psi.PsiMethod;
+import com.intellij.psi.PsiMethodCallExpression;
+import com.intellij.psi.PsiReferenceExpression;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.IOException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class extends the LocalInspectionTool to check for the use of connection strings in the code.
+ * If the method is called from an Azure client class, a problem is registered with the suggestion message.
+ */
+public class ConnectionStringCheck extends LocalInspectionTool {
+
+
+    /**
+     * This method builds the visitor for the inspection tool.
+     * @param holder ProblemsHolder to register problems
+     * @param isOnTheFly boolean to check if the inspection is on the fly. If true, the inspection is performed as you type.
+     * @return
+     */
```

Comment:
Javadoc can be updated to include return type and make description concise.
   ```java
    /**
     * Builds the visitor for the inspection tool.
     *
     * @param holder ProblemsHolder to register problems
     * @param isOnTheFly boolean to check if the inspection is on the fly. If true, the inspection is performed as you type.
     * @return the visitor for inspecting elements
     */
     ```

Reply:
done:)